test/runtests.jl

```diff
@@ -226,6 +226,49 @@ else
 end # @static if VERSION < v"1.8"
     end
 
+    @testset "FIFOLock" begin
+@static if VERSION < v"1.10"
```

Comment:
It probably doesn't matter, but for `<` on versions, i think you're supposed to do `< v"1.10-"`, because otherwise someone using a prerelease (e.g. `v"1.10.0-rc1"`) would still hit this block:
```julia
julia> v"v1.10.0-rc2" < v"1.10.0"
true

julia> v"v1.10.0-rc2" < v"1.10.0-"
false
```

Reply:
I guess this probably applies to the `>= v"1.10"` you have at the top as well.. 🤔

Reply:
Fixed both places.